package.json

```diff
@@ -39,5 +43,15 @@
     "karma-junit-reporter": "^1.2.0",
     "karma-phantomjs-launcher": "^1.0.2",
     "karma-requirejs": "^1.1.0"
-  }
+  },
+  "keywords": [
```

Comment:
OK, they're added now, and in lowercase. Do you think we should leave out "javascript"? It may be redundant as a search keyword in npm.

I also added "browser". Package authors tend to include that so that packages that are meant to be executed in a browser can be distinguished from the ones meant for NodeJS runtime:

https://www.npmjs.com/search?q=keywords:browser